codec/packer.go

```diff
@@ -81,6 +81,17 @@ func (p *Packer) UnpackAddress(dest *Address) {
 	}
 }
 
+func (p *Packer) PackActionID(a LID) {
+	p.p.PackFixedBytes(a[:])
+}
+
+func (p *Packer) UnpackActionID(required bool, dest *LID) {
+	copy((*dest)[:], p.p.UnpackFixedBytes(AddressLen))
+	if required && *dest == EmptyAddress {
+		p.addErr(fmt.Errorf("%w: LID field is not populated", ErrFieldNotPopulated))
+	}
+}
+
```

Comment:
rename to use `LID` over `ActionID`. and also, is there a way to combine with `PackAddress` and `UnpackAddress`?

Reply:
This is unneeded. Just always use the LID constant.

Even if you did want to do this, you probably wouldn't use a generic.